### PR TITLE
propose RFC 0024: DIDCom over XMPP

### PR DESCRIPTION
**Pull request by Oskar-van-Deventer**
Signed-off-by: Oskar van Deventer oskar.vandeventer@tno.nl

### CODE REVIEW

File: code/generate_index.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
 import os
 import re
+from operator import itemgetter
```
Comment (dhh1128) [inline code/generate_index.py]: I don't think changes from this file should be part of the PR. I think we accidentally picked up changes from a different PR.

File: concepts/0021-didcomm-message-anatomy/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# 0021: DID Comm Message Anatomy
```
Comment (dhh1128) [inline concepts/0021-didcomm-message-anatomy/README.md]: I don't think changes from this file should be part of this PR; it looks like we're mixing 2 different concerns.

**Comment by dhh1128** [comment]
I think this could be merged if we can remove changes to some files that are not part of this topic, that accidentally crept into the PR>

File: concepts/0021-didcomm-message-anatomy/README.md (hunk 2 above)
Comment (kdenhartog) [reply concepts/0021-didcomm-message-anatomy/README.md]: Second this. Can you remove these changes as well as the changes made to the code to a separate PR?

File: code/generate_index.py (hunk 1 above)
Comment (Oskar-van-Deventer) [reply code/generate_index.py]: How do I remove that accidentally added stuff? How can I clean it up? Or can the poluter clean this up, please?

File: concepts/0021-didcomm-message-anatomy/README.md (hunk 2 above)
Comment (Oskar-van-Deventer) [reply concepts/0021-didcomm-message-anatomy/README.md]: How do I remove this polution? Can Daniel or Kyle remove it? Can the poluter remove it?

Comment (dhh1128) [reply concepts/0021-didcomm-message-anatomy/README.md]: @Oskar-van-Deventer: I will clean up the extra stuff and fix the DCO problem. Please give me until the end of my workday. Then we can merge the PR.

**Comment by dhh1128** [comment]
I have replaced this with a clean PR that didn't incorporate spurious updates from the master branch. See #83. That PR has now been merged, so I'm closing this one.

**Comment by Oskar-van-Deventer** [comment]
Daniel, 
Excellent, thank you for your help!
Oskar

**Pull request closed without merging**